iOS/SampleApp/SampleApp/Services/ManagingInformationAlerts.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+extension ViewController {
```

Comment:
The same applies to this class - we don't need to handle any alerts when returning to sample app, it doesn't need to support anything else than opening FVM and printing FVM outcome to console

Reply:
I want to allow end-user to customize main Screen Text and damaged car parts for themselves before go to FVM

Reply:
All right, the question is, as we don't need it actually (because of possibility to hard-code these values), if we want to merge this to master. The only way we can employ this is during demo, isn't it?

Reply:
Probably yes, but I don't know at this moment. Actually, I have already finished writing SampleApp and it is prepared to demo.